install/0000_00_cluster-version-operator_02_usc_rbac-DevPreviewNoUpgrade.yaml

```diff
@@ -0,0 +1,165 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: update-status-controller-library
+  namespace: openshift-cluster-version
+  annotations:
+    kubernetes.io/description: Role that allows the USC binary to operate local resources needed by the o/library-go controller code
```

Comment:
nit: `o/library-go` -> `openshift/library-go` or drop the library reference entirely, so that cluster admins reading this are less likely to be confused by the `o/` shorthand?

Reply:
Good point

Reply:
Resolved in 0d2d67a3866773f64aa7c0b60a4a3ccc287a2621